testsuite/MathMLToDisplay/General/Colors/issue1002.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+  <head>
+    <title>Test SVG output inherting color</title>
+    <!-- Copyright (c) 2015 MathJax Consortium
+         License: Apache License 2.0 -->
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <script type="text/javascript" src="../../../header.js"></script>
+    <script type="text/javascript" src="../../../scriptTests.js"></script>
+    <script type="text/javascript">
+      function getTestCases() {
+      var path = document.querySelectorAll('g')[0];
+      var check1 = (path.getAttribute('fill') === 'currentColor' & path.getAttribute('stroke') === 'currentColor');
+      var computedColor = window.getComputedStyle(path).color;
```

Comment:
I didn't repeat my comment about it being OK in this case because it already requires SVG.  I assumed the comment above and the "similarly to above" would cover that.